search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+let currentTrend = '';
+let currentCustomSearchData = '';
+
+/** Returns current trend user is viewing. */
+function getCurrentTrend() {
+  return currentTrend;
+}
+
+/** Returns current custom search data for trend user is viewing. */
+function getCurrentCustomSearchData() {
+  return currentCustomSearchData;
+}
+
+/** 
+ * Retrieves relevant data for new trend and reconstructs map with new data.
+ * @param {string} trend New trend to get data for.
+ */
+// TODO(carmenbenitez): Uncomment if/else block to show data for other search
+// results when custom search data for all top trends set up. Replace top trend
+// in else block with top trend.
+function setNewTrend(trend) {
+  // if (trend != null){
+  //   currentTrend = trend;
+  // } else {
+  //   currentTrend = 'TOP TREND HERE';
+  // }
+
+  currentTrend = 'The Old Guard';
```

Comment:
@carmenBen needs this for implementing the click feature, can you leave a todo for yourself here Carmen?

Reply:
So there is already a TODO for this function. The reason I am not implementing this yet(and neither should your branch Chenyu) is that we have stopped fetching new custom search data, but I believe the trends data is still being updated, meaning this will list new trends that we do not have data for. If we have stopped fetching new trends then yes we could implement this.